src/main/scala/com/databricks/spark/csv/CsvRelation.scala

```diff
@@ -17,18 +17,19 @@ package com.databricks.spark.csv
 
 import java.io.IOException
 
-import scala.collection.JavaConversions._
-import scala.util.control.NonFatal
-
+import com.ayasdi.bigdf.readers._
+import com.databricks.spark.csv.util.ParseModes
 import org.apache.commons.csv._
 import org.apache.hadoop.fs.Path
+import org.apache.spark.rdd.RDD
 import org.apache.spark.sql._
 import org.apache.spark.sql.catalyst.expressions._
 import org.apache.spark.sql.sources.{BaseRelation, InsertableRelation, TableScan}
-import org.apache.spark.sql.types.{StructType, StructField, StringType}
+import org.apache.spark.sql.types.{StringType, StructField, StructType}
 import org.slf4j.LoggerFactory
 
-import com.databricks.spark.csv.util.ParseModes
+import scala.collection.JavaConversions._
+import scala.util.control.NonFatal
```

Comment:
actually that plugin is deprecated because IntelliJ 13 or so added intrinsic support. See the style guide for the configuration for IntelliJ's thing: https://github.com/databricks/scala-style-guide#imports